index.html

```diff
@@ -1,377 +1,312 @@
-    crossorigin="anonymous"></script>
+            .carousel-item img {
+                object-fit: cover;
+                height: 100vh;
+            }
 
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Cabin:ital,wght@0,400;1,500&family=Open+Sans:ital,wght@0,300;0,400;1,500&family=Sora:wght@500&display=swap" rel="stylesheet">
-</head>
+            .carousel-caption {
+                background-color: rgba(0, 0, 0, 0.5);
+                border-radius: 2.5rem;
+                padding: 1rem;
+            }
 
-<body>
-  <div class="row" id="Topnav" style="overflow-x:hidden; ">
-    <div class="col-12">
-      <div class="topnav" id="myTopnav">
-        <img class="xxxx" src="shreetransparent.png" alt="Logo">
-        <div class="Nbar">
-          <a href="index.html" class="active">INTRO</a>
-          <a href="spas.html" style="padding-left: 35px;">OUR PRODUCTS</a>
-          <a href="contact.html" style="padding-left: 30px;">CONTACT US</a>
-        </div>
-        <a href="javascript:void(0);" class="icon" onclick="myFunction()">
-          <i class="fa fa-bars"></i>
-        </a>
+            #about {
+                background-image: url(https://raw.githubusercontent.com/Meeha66/images/main/aboutusbg1.jpg);
+                background-size: cover;
+                background-repeat: no-repeat;
+            }
+
+            #about .col {
+                padding: 1.5rem;
+            }
+
+            footer .col {
+                padding: 1rem;
+            }
 
-      </div>
-    </div>
-  </div>
-  <main class="scroll-container">
-    <section>
-  <div class="row" style="height: 100vh;">
-    <div class="col-12">
-      <div class="">
-        <main>
-          <div id="da-slider" class="da-slider">
-            <div id="myCarousel" class="carousel slide" data-bs-ride="carousel">
-              <div class="carousel-indicators">
-                <button type="button" data-bs-target="#myCarousel" data-bs-slide-to="0" class="active"
-                  aria-current="true" aria-label="Slide 1"></button>
-                <button type="button" data-bs-target="#myCarousel" data-bs-slide-to="1" aria-label="Slide 2"></button>
-                <button type="button" data-bs-target="#myCarousel" data-bs-slide-to="2" aria-label="Slide 3"></button>
-                <button type="button" data-bs-target="#myCarousel" data-bs-slide-to="3" aria-label="Slide 4"></button>
-                <button type="button" data-bs-target="#myCarousel" data-bs-slide-to="4" aria-label="Slide 5"></button>
-              </div>
-              <div class="carousel-inner" style="width: 100%;
-          height: 100vh;">
+            #about p {
+                font-size: 115%;
+                text-align: justify;
+                margin: 0;
+            }
+        </style>
```

Comment:
These are the only custom CSS rules I've used, you can move them to an external file. Everything else uses bootstrap.